### PR TITLE
Backfill stockreport server date

### PR DESCRIPTION
**Pull request by dannyroberts**
@czue cc @NoahCarnahan 

If someone can just sign off on this (and not merge) I can coordinate the merging and running on
- [x] staging
- [x] production
- [x] india

### CODE REVIEW

**Comment by emord** [comment]
Doing this on staging, might be tricky. This was applied after I had deployed Django 1.7 to staging

https://github.com/dimagi/commcare-hq/commit/325886fd9ce210cbb7a9b9a9d1a10cfb07b05e88 

Not sure how south will react adding a column that already exists, if you try to apply it with south. Could this be done with a management command?

File: corehq/ex-submodules/casexml/apps/stock/south_migrations/0016_backfill_stockreport_server_date.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+             .update(server_date=server_date))
+    blank_count_after = StockReport.objects.filter(server_date=None).count()
+    return BackfillResult(
+        problem_form_ids=problem_form_ids,
+        blank_count_before=blank_count_before,
+        blank_count_after=blank_count_after,
+    )
+
+
+class FooEx(Exception):
```
Comment (czue) [inline corehq/ex-submodules/casexml/apps/stock/south_migrations/0016_backfill_stockreport_server_date.py]: remove?

**Comment by czue** [comment]
looks good. is the idea that you'll do one pass of sofabed models and then see how it goes and whether it's necessary to do one that goes directly to couch due to missing/bad/weird data?

**Comment by dannyroberts** [comment]
Yeah. If it's a systematic thing that forms are missing then I'll have to do another pass. If it's a few random things that don't even look like form IDs (I've seen other stuff inexplicably in the form_id field), and one or two error forms or something, then probably not worth it.

File: corehq/ex-submodules/casexml/apps/stock/south_migrations/0016_backfill_stockreport_server_date.py (hunk 1 above)
Comment (dannyroberts) [inline corehq/ex-submodules/casexml/apps/stock/south_migrations/0016_backfill_stockreport_server_date.py]: Oops

**Comment by czue** [comment]
cool, i'm signed off whenever you want to lead

**Pull request merged**